networkx/algorithms/cycles.py

```diff
@@ -106,7 +106,7 @@ def cycle_basis(G, root=None):
 def simple_cycles(G, length_bound=None):
     """Find simple cycles (elementary circuits) of a graph.
 
-    A `simple cycle`, or `elementary circuit`, is a closed path where
+    A ``simple cycle``, or ``elementary circuit``, is a closed path where
```

Comment:
Should these quotes be "simple cycle" instead of ``simple cycle``? I was thinking that double backticks was for code, but I often get that mixed around... I'm fine either way.